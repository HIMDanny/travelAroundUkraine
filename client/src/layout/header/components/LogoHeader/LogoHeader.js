import React from 'react';
import { NavLink } from 'react-router-dom';

const LogoHeader = () => (
  <NavLink to="/" style={{ textDecoration: 'none' }}>
    <img src="/assets/images/logo/logo.png" alt="logo" width="65" height="50" />
  </NavLink>
);

export default LogoHeader;
