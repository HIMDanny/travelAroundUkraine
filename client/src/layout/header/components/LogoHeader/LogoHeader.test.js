import { render } from '@testing-library/react';
import LogoHeader from './LogoHeader';
import { BrowserRouter } from 'react-router-dom';

describe('Render LogoHeader', () => {
  test('should LogoHeader render', () => {
    const { asFragment } = render(
      <BrowserRouter>
        <LogoHeader />
      </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
