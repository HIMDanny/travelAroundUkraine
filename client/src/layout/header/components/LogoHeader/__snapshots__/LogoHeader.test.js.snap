// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render LogoHeader should LogoHeader render 1`] = `
<DocumentFragment>
  <a
    aria-current="page"
    class="active"
    href="/"
    style="text-decoration: none;"
  >
    <img
      alt="logo"
      height="60"
      src="/assets/images/logo/logo.png"
      width="75"
    />
  </a>
</DocumentFragment>
`;
